docs/pages/concept/web.md

```diff
@@ -0,0 +1,145 @@
+    "url": "https://example.com",
+    "resolution": {
+        "width": 1920,
+        "height": 1080
+    }
+}
+```
+- `instance_id` - unique renderer identifier, which is used to refer to the web renderer.
+- `url` - website url. All protocols supported by chromium can be used
+
+Now, we can define scene with web view component:
+```typescript
+{
+    "type": "update_scene",
+    "outputs": [
+        {
+            "output_id": "output_1",
+            "root": {
+                "id": "embed_input_on_website",
+                "type": "web_view",
+                "instance_id": "example_website",
+            }
+        }
+    ]
+}
+```
+
+`instance_id` - the ID of previously registered web renderer.
+
+That's it!
+
+:::warning
+One specific web renderer instance can be used only by one component!
+:::
+
+
+## Embedding other components
+Embedding is a process of rendering children components on a website.
+
+There are 3 embedding methods available:
+- `chromium_embedding` - the most flexible method. The children are passed directly to the chromium instance and rendered on HTML canvas. You can animate them and use CSS styles without any problems. However, while it offers the best flexibility, it's also the slowest method.
+- `native_embedding_over_content` - renders the website first and then renders child components on top of the website. It's faster than `chromium_embedding`, but CSS styling does not work.
+- `native_embedding_under_content` - renders component children first and then the website on top of them. The website needs a transparent background. Performance and limitations are the same as for `native_embedding_over_content`.
+
+The default embedding method is `native_embedding_over_content`.
+You can change the embedding method in the [register renderer request](../api/routes#register-renderer). For example:
+```typescript
+{
+    "type": "register",
+    "entity_type": "web_renderer",
+    "instance_id": "example_website",
+    "url": "https://example.com",
+    "resolution": {
+        "width": 1920,
+        "height": 1080
+    },
+    // NEW
+    "embedding_method": "chromium_embedding"
+}
+```
+
+
+To embed child components into the website we need to modify the scene definition, as well as the website.
+Let's update the scene definition and embed an input stream component into the website:
+```typescript
+{
+    "type": "update_scene",
+    "outputs": [
+        {
+            "output_id": "output_1",
+            "root": {
+                "id": "embed_input_on_website",
+                "type": "web_view",
+                "instance_id": "example_website",
+                // NEW
+                "children": [
+                    {
+                        "type": "input_stream",
+                        "input_id": "input_1",
+                    }
+                ]
+            }
+        }
+    ],
+}
+```
+
+:::note
+The input stream has to be registered beforehand.
+:::
+
+Now we need to modify the website:
```

Comment:
do not just provide snippets of code for specific case, explain what user needs to do in general. e.g. It's not obvious what would need to be done to make it work e.g. in react website